### PR TITLE
Model::toArray use of getters

### PR DESCRIPTION
**Pull request by rudiservo**
Added a short explanation of Model:toArray use of getters

### CODE REVIEW

File: docs/db-models.md [hunk 1]
```diff
@@ -1060,6 +1061,60 @@ print_r(
 //      'inv_title'       = 'Invoice for ACME Inc.',
 //      'inv_total'       = 100,
 //  ]
+
+```
+
+toArray uses getters by default, to disable set $useGetters to false
```
Comment (niden) [inline docs/db-models.md]: `toArray` uses getters by default, to disable this behavior, set `$useGetters` to `false`

**Review by niden — CHANGES_REQUESTED** [review]
One minor change please.

File: docs/db-models.md (hunk 1 above)
Comment (rudiservo) [reply docs/db-models.md]: Done

**Review by niden — APPROVED** [review]

**Comment by niden** [comment]
Thank you @rudiservo

**Pull request merged**